src/ResourceManager/ContainerRegistry/Commands.ContainerRegistry/help/Get-AzureRmContainerRegistryWebhookEvent.md

```diff
@@ -0,0 +1,157 @@
+---
+external help file: Microsoft.Azure.Commands.ContainerRegistry.dll-Help.xml
+Module Name: AzureRM.ContainerRegistry
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.containerregistry/get-azurermcontainerregistrycredential
+schema: 2.0.0
+---
+
+# Get-AzureRmContainerRegistryWebhookEvent
+
+## SYNOPSIS
+Get events of a container registry webhook.
+
+## SYNTAX
+
+### ListWebhookEventsByNameResourceGroupParameterSet
+```
+Get-AzureRmContainerRegistryWebhookEvent [-Name] <String> [-ResourceGroupName] <String>
+ [-RegistryName] <String> [-DefaultProfile <IAzureContextContainer>]
+```
+
+### ListWebhookEventsByWebhookObjectParameterSet
+```
+Get-AzureRmContainerRegistryWebhookEvent -Webhook <PSContainerRegistryWebhook>
+ [-DefaultProfile <IAzureContextContainer>]
+```
+
+### ResourceIdParameterSet
+```
+Get-AzureRmContainerRegistryWebhookEvent -ResourceId <String> [-DefaultProfile <IAzureContextContainer>]
+```
+
+## DESCRIPTION
+The Get-AzureRmContainerRegistryWebhookEvent cmdlet lists all the events of a webhook.
+
+## EXAMPLES
+
+### Example 1: Get all the events of a webhook.
+```powershell
+PS C:\>Get-AzureRmContainerRegistryWebhookEvent -ResourceGroupName mattacrtest001 -RegistryName premium001 -Name webhook001
+
+Reque Request Headers      Respo Response Headers     Response Content
+st Me                      nse S
+thod                       tatus
+                           Code
+----- ---------------      ----- ----------------     ----------------
+POST  {[SpecialHeader, ... 200   {[Vary, Accept-En... <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"...
+POST  {[SpecialHeader, ... 200   {[Vary, Accept-En... <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"...
+POST  {[SpecialHeader, ... 200   {[Vary, Accept-En... <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"...
```

Comment:
Can you please improve how the output is printed and also use some meaningful examples? In general users would like to see the event notification payload and what the response is. Please also refer to Azure Portal or Az CLI to see which properties are highlighted.